lib/pbench/test/unit/server/test_schema.py

```diff
@@ -68,21 +82,27 @@ def ok(auth: Auth, username: str) -> User:
             (ParamType.DATE, 1),  # not a string representing a date
             (ParamType.USER, "drb"),  # we haven't established a "drb" user
             (ParamType.USER, False),  # not a user string
-            (ParamType.ACCESS, "foobar"),  # ACCESS is "public" or "private"
+            (ParamType.ACCESS, ["foobar"]),  # ACCESS is "public" or "private"
             (ParamType.ACCESS, 0),  # ACCESS must be a string
-            (ParamType.LIST, "foobar"),  # Not a list
-            (ParamType.LIST, 0),  # Not a list
         ),
     )
     def test_failed_conversions(self, test, monkeypatch):
+        """
+        Test unsuccessful parameter conversion / normalization.
+
+        NOTE that we can't test KEYWORD here without a Parameter to define the
+        set of keywords; we'll test that separately.
```

Comment:
Should this mention `LIST`, like line 53 does?

Reply:
Yeah, another stale comment. I hate comments: it's too easy to miss stuff when changing code. 🐛